### ISSUE TITLE
We Need a Logo

### ISSUE THREAD

**Issue by mzagaja** [codeforboston/cambridge_energy_app#45]
If someone is artsy and wants to create an EnerSave Logo that'd be awesome.

**Comment by cmajel** [comment]
@mzagaja I'm a designer who has visited code for boston before. Would be happy to help with a few logo concepts. 

The questions I typically ask are below. I'm cool coming to the meeting up tomorrow to chat or we can do it over github:
- confirm the spelling + casing should be: EnerSave (not enersave or Ener Save)
- what forms do you need the logo in? (obviously the site, but favicon? app icon? need a mobile version? I can work with whatever you already have set up)
- what do you want EnerSave to convey? (i.e. community, savings, environmental benefits, energy sources, all the above?) Hearing your 2 sentence explanation for why someone would use the app would be helpful.
- are you locked into any particular design choices already (fonts, coloring) or is this a blank slate?
- any examples of logos you like or dislike
- any specific ideas you have when you think of the logo. 
- timeline for when you would like drafts and a final logo.

**Comment by mzagaja** [comment]
Thanks so much for offering @cmajel! Feel free to drop in at the Meetup tonight if you want to discuss, but I am going to go over this tonight either way and will memorialize the answers here.

**Comment by cmajel** [comment]
@mzagaja awesome! i work down the street so I'll stop by tonight.

**Comment by cmajel** [comment]
@mzagaja I am still playing around with a few ideas but wanted to share what I have so far. I can stop by tonight for a little bit and get your thoughts - may have another 1 or 2 to add to the pile by then.

![es_logo_ideas_03012016](https://cloud.githubusercontent.com/assets/2374206/13426733/6ab41528-df7d-11e5-88ae-41402e3b2855.png)

**Comment by cmajel** [comment]
I did a round 2 this weekend - gonna put everything here.

Results from the survey- E was the favorite, B the second favorite: https://docs.google.com/spreadsheets/d/1WOAqD8qdGK36EKm-Oa5aVjvYcbUngaUHq39E3fgHILU/edit?usp=sharing

I did a few round on E and B, and I also explored the atom suggestion I got last week. I am liking E more and more - it's simple, has good white space, and works well on a dark background. I did do a atom/knot style, which I like, but it feels a bit more formal. I think E (particularly E1 with the color adjustments) would work the best.

Comps below. I created another survey to get feedback on this round and then I'll send over the finals. 
Survey: https://www.surveymonkey.com/r/K3RFPDH

![round2_logoe](https://cloud.githubusercontent.com/assets/2374206/13603417/23f13bfa-e50b-11e5-9715-5a572c2efb23.png)
![round2_misc](https://cloud.githubusercontent.com/assets/2374206/13603416/23ebed80-e50b-11e5-959a-49a19028812d.png)
